services/wallet/inputs.go

```diff
@@ -297,6 +298,15 @@ func (r *ZebPayLinkingRequest) Decode(ctx context.Context, v []byte) error {
 
 // HandleErrorsZebPay returns an AppError for the given err.
 func HandleErrorsZebPay(err error) *handlers.AppError {
+	// KYC issue
+	if errors.Is(err, errZPInvalidKyc) {
```

Comment:
I might be missing it but does the `errZPInvalidKyc` not get returned by the [service linking call](https://github.com/brave-intl/bat-go/blob/da15b7cef7af7faa10a343ce5c12c63b2de882a7/services/wallet/controllers_v3.go#L254) and so this wont function wont get called

Reply:
You're right. It's only called for validation errors:

https://github.com/brave-intl/bat-go/blob/da15b7cef7af7faa10a343ce5c12c63b2de882a7/services/wallet/controllers_v3.go#L250-L252

Reply:
then how does my test work?

Reply:
i believe the linking_info jwt is decoded/validated from the inputs, so i believe this does work as expected.